app/views/arrangement/change_day.scala.html

```diff
@@ -58,8 +58,14 @@ <h1 class="heading-xlarge">@Messages("ssttp.arrangement.change_day.title")</h1>
             <label for="other">@Messages("ssttp.arrangement.changeday.different")</label>
         </div>
 
-        <div id="customDate" class="panel panel-border-narrow js-hidden form-group">
+        <div id="customDate" class="panel panel-border-narrow js-hidden form-group @if(dayForm.hasErrors){form-field--error}">
             <label for="dayInput" >@Messages("ssttp.arrangement.changeday.enter.date")</label>
+            @if(dayForm.hasErrors) {
+                <br>
```

Comment:
can we use `<br>` tags in html? It should be done using css, should it?

Reply:
just asking

Reply:
There was already one on line 69. They seem to work as anticipated. Might be a question for Ola.